tests/handles.test.ts

```diff
@@ -0,0 +1,126 @@
+import { ControllerFactory } from './controllers';
+import { DatabaseConnection, PortalState, UserFactory } from './data';
+
+beforeAll(async () => {
```

Comment:
nit: let's rename the file to `user.handles.test.ts` again to make the context clear of what "handles" refer to.